table/meta_blocks.cc

```diff
@@ -463,11 +463,10 @@ Status FindMetaBlock(InternalIterator* meta_index_iter,
   }
 }
 
-Status FindMetaBlockInFile(RandomAccessFileReader* file, uint64_t file_size,
+Status ReadMetaBlockInFile(RandomAccessFileReader* file, uint64_t file_size,
```

Comment:
Should it be `ReadMetaIndexBlockInFile()`? IIUC this is a distinct block from what `FindMetaBlockInFile()` and `ReadMetaBlock()` could return.